tcf_website/templates/course/course.html

```diff
@@ -34,6 +34,11 @@ <h2>{{ course.title }}</h2>
             <br/>
             <br/>
 
+            <!-- Unsure if we want to remove it when they are looking at all semesters. If so, may need the opposite of "old" class. -->
+            {% if not this_semester_instructors %}
+                <h5 class="">Looks like this course isn't taught this semester. Sort by "All" in the top right to see previous semesters.</div>
```

Comment:
Might I recommend rephrasing this to "Looks like this course isn't being taught this semester."?